client/debug/main.go

```diff
@@ -0,0 +1,197 @@
+package debug
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+)
+
+func Cmd(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "debug",
+		Short: "tool for helping with debugging your application",
+		RunE:  client.ValidateCmd,
+	}
+
+	cmd.AddCommand(PubkeyCmd(cdc))
+	cmd.AddCommand(AddrCmd())
+	cmd.AddCommand(RawBytesCmd())
+
+	return cmd
+}
+
```

Comment:
All the indentations for the `Long` descriptions are off. Please test this against Gaia.

Reply:
seems trimspace wasn't working with a multiline string.